### PR TITLE
Playable on Linux

### PR DESCRIPTION
**Pull request by naddoska**
Hey :)
I was trying to do the Game Over Sound, unfortunately I use Linux and it did not work on Linux, so I made it work on Linux. 
Maybe you like that. 

Cool game btw. 

:)

### CODE REVIEW

**Comment by naddoska** [comment]
actually this should also work on MAC but i cant test that :D

File: config.py [hunk 1]
```diff
@@ -16,9 +16,11 @@ def __init__(self):
...
             os.path.join('assets', 'graphics', 'background-black-wide.png')))
 
-        windows_user = ctypes.windll.user32
-        self.monitor_size = (windows_user.GetSystemMetrics(0),
-                             windows_user.GetSystemMetrics(1))
+        self.CANVAS = pygame.display.set_mode(
+            (self.WIDTH, self.HEIGHT), pygame.RESIZABLE)
+
+        self.monitor_size = pygame.display.Info().current_w, pygame.display.Info().current_h
+        # print(pygame.display.Info())
```
Comment (nkitsaini) [inline config.py:23]: Can we remove this comment

**Review by nkitsaini — CHANGES_REQUESTED** [review]

File: config.py (hunk 1 above)
Comment (naddoska) [reply config.py:23]: Hey! The comment was in the code before and I actually used it to make the change so i was thinking maybe its there for when people try to change something within the code, that is why i left it for now. I don't really know if that was the initial intention of the creator :D

Comment (Mayank0255) [reply config.py:23]: @naddoska can remove the comment, I must have forgotten about it. @nkitsaini thanks for pointing out

**Comment by Mayank0255** [comment]
@naddoska now the game would be playable on linux if we run the python script through terminal right?

as I believe the build files are of different extension  for linux and mac. `.exe` won't be supported in it

File: config.py (hunk 1 above)
Comment (naddoska) [reply config.py]: @Mayank0255  I fixed it :)

**Comment by naddoska** [comment]
Yeah, as of right now you would have to run it through the terminal.

**Review by sharkwouter — APPROVED** [review]
This looks good to me. I've tested this on Ubuntu 22.04.

**Comment by Mayank0255** [comment]
@naddoska 
Sorry for the delay, got busy with work.

Can you pls update the readme with the command that linux and mac users will have to run on terminal?

**Comment by sharkwouter** [comment]
@Mayank0255 what I did was just use the run command found in the README.

**Review by Mayank0255 — APPROVED** [review]

**Pull request merged**